src/parseTools.js

```diff
@@ -429,7 +429,16 @@ function makeSetValueImpl(ptr, pos, value, type) {
 function makeHEAPView(which, start, end) {
   const size = parseInt(which.replace('U', '').replace('F', '')) / 8;
   const shift = Math.log2(size);
-  const mod = size == 1 ? '' : (CAN_ADDRESS_2GB || MEMORY64) ? ('>>>' + shift) : ('>>' + shift);
+  let mod = '';
+  if (size) {
```

Comment:
```suggestion
  if (shift) {
```
When the shift is 0 (hence size is 1) we want to add no modifier IIUC.

lgtm otherwise.

Reply:
oops, good catch.